src/utils/getTextForLanguage.js

```diff
@@ -8,8 +8,8 @@ import { interpolateSnippets } from './interpolation.js';
 import { localizations } from '../localization/all';
 import inlineComponents from '../localization/inlineComponents';
 
-// interpolate components into English snippets:
-const interpolatedEnglish = interpolateSnippets(localizations.en, inlineComponents);
+// store interpolated and (if necessary) merged snippets objects
```

Comment:
Note for further discussion: Would it be worth making this a jsdoc documented variable?